medical_lab/models/medical_lab.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLab(models.Model):
+    _name = "medical.lab"
+    _description = "Medical Labs"
+
+    name = fields.Char(
+        string='ID',
+        size=128,
+        required=True,
+        default=lambda s: s.env['ir.sequence'].next_by_code(
+            'medical.lab',
+        ),
+        help='Unique identifier for lab.',
+    )
+    test_type_id = fields.Many2one(
+        string='Test Type',
+        comodel_name='medical.lab.test.type',
+        help='Lab test type.',
+    )
+    physician_id = fields.Many2one(
+        string='Pathologist',
```

Comment:
Either change to physician, or change **request_physician_id** string attribute to 'requesting pathologist' instead of requesting physician.